src/document/types.ml

```diff
@@ -85,11 +85,25 @@ and Block : sig
     | Math of Math.t
     | Verbatim of string
     | Raw_markup of Raw_markup.t
+    | Table_light of Table_light.t
+    | Table_heavy of Table_heavy.t
```

Comment:
Why do we need the two kinds in the document ? It seems like both syntaxes can be compiled to the same representation.

Reply:
done